wp-content/themes/citylimits/inc/enqueue.php

```diff
@@ -45,9 +45,9 @@ function largo_enqueue_js() {
 
 		wp_enqueue_style(
 			'largo-child-styles',
-			get_stylesheet_directory_uri() . '/style.css',
+			get_stylesheet_directory_uri() . '/child-style.css',
```

Comment:
Moving this file from `/style.css` to `/child-style.css` will probably break your theme, as `style.css` is the _one_ file in a child theme that must be present. If you want the output of Grunt's LESS compiling to be the file that's enqueued, there are a couple options:

## Separating `style.css` from the Grunt output

- Keep `style.css` for just its header. Make sure it's not enqueued, by using remove_action on this function: https://github.com/INN/largo/blob/c41e28e6381397be1918184b906ec84bc019745b/inc/enqueue.php#L124-L144
- Have the Grunt output go to `/child-style.css` or `/css/child-style.css`

## Merging the Grunt output and `style.css`

- Edit the Gruntfile to make the `grunt less` task output to `style.css`.
- Make sure that the generated file keeps the required metadata at the top of the file.

Reply:
I renamed the file because of [https://github.com/INN/Largo-Sample-Child-Theme/issues/14](url). But okay: I've followed your first suggestion and kept `style.css` just for its header; and I've unenqueued it as well.

Reply:
That issue makes sense.